contracts/auctions/src/storage.rs

```diff
@@ -52,7 +48,7 @@ pub struct Auction {
 #[contracttype]
 pub struct HighestBid {
     pub bid: u64,
-    pub bidder: Address,
+    pub bidder: Option<Address>,
 }
```

Comment:
Where is this coming from?

Reply:
previously we weren't allowed to have `Option<Address>` in structs for the SDK, but now this works. 

Having it as `Option` allows removing this piece of code, that I don't like.

```rust
pub fn get_highest_bid(env: &Env, auction_id: u64) -> Result<HighestBid, ContractError> {
    let highest_bid = env
        .storage()
        .instance()
        .get(&DataKey::HighestBid(auction_id))
        .unwrap_or(HighestBid {
            bid: 0,
            // I know
            bidder: get_admin_old(env)?,
        });
}
```

https://github.com/Phoenix-Protocol-Group/phoenix-nft-marketplace/blob/06987f77166ca6a403923eacc1a0f10ee667cb3a/contracts/auctions/src/storage.rs#L244-L253

it's the only reason for the change in the struct

Reply:
That's alright. 
Next time make a separate commit for this and don't hide it completely under a different major significant change.

Reply:
reverted in 0bb025f6cab3